src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/GeneratorTests.Helpers.cs

```diff
@@ -138,7 +138,11 @@ private static async Task VerifyThatSourceIsGenerated(string testSourceCode)
                 string source = string.Join(Environment.NewLine, lines).TrimEnd(Environment.NewLine.ToCharArray()) + Environment.NewLine;
                 path = Path.Combine($"{repoRootDir}\\src\\libraries\\Microsoft.Extensions.Configuration.Binder\\tests\\SourceGenerationTests\\", path);
 
-                await File.WriteAllTextAsync(path, source).ConfigureAwait(false);
+#if NETCOREAPP
+                await File.WriteAllTextAsync(path, source);
```

Comment:
>File.WriteAllTextAsync(path, source); [](http://example.com/codeflow?start=22&length=37)

I think you still need `.ConfigureAwait(false);`

Reply:
It's test code only run by xunit, so not really necessary.